shell/browser/ui/file_dialog_linux.cc

```diff
@@ -44,14 +44,14 @@ ui::SelectFileDialog::FileTypeInfo GetFilterInfo(const Filters& filters) {
   ui::SelectFileDialog::FileTypeInfo file_type_info;
 
   for (const auto& [name, extension_group] : filters) {
-    file_type_info.extension_description_overrides.push_back(
-        base::UTF8ToUTF16(name));
-
     const bool has_all_files_wildcard = std::ranges::any_of(
-        extension_group, [](const auto& ext) { return ext == "*"; });
-    if (has_all_files_wildcard) {
+      extension_group, [](const auto& ext) { return ext == "*"; });
+
+      if (has_all_files_wildcard) {
```

Comment:
One indentation too much here, I guess

Reply:
if you could back out this newline change @razzeee we can get this merged !

Reply:
Fixed thanks